internal/class.h

```diff
@@ -146,14 +145,18 @@ struct RClass {
 
 // Assert that classes can be embedded in heaps[2] (which has 160B slot size)
 // On 32bit platforms there is no variable width allocation so it doesn't matter.
-// TODO: restore this assertion after shrinking rb_classext_t
-// STATIC_ASSERT(sizeof_rb_classext_t, sizeof(struct RClass) + sizeof(rb_classext_t) <= 4 * RVALUE_SIZE || SIZEOF_VALUE < SIZEOF_LONG_LONG);
+STATIC_ASSERT(sizeof_rb_classext_t, sizeof(struct RClass) + sizeof(rb_classext_t) <= 4 * RVALUE_SIZE || SIZEOF_VALUE < SIZEOF_LONG_LONG);
```

Comment:
This is safer than adding `sizeof`s.
Of course, after `struct RClass_and_rb_classext_t` declaration.

```suggestion
#if SIZEOF_VALUE >= SIZEOF_LONG_LONG
STATIC_ASSERT(sizeof_RClass_and_rb_classext_t, sizeof(struct RClass_and_rb_classext_t) <= 4 * RVALUE_SIZE);
#endif
```

Reply:
Done. Thank you @nobu!